tests/unit/widget-checkbox.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'chai';
+import { mount, shallowMount } from '@vue/test-utils';
+import Vue from 'vue';
+import WidgetCheckbox from '@/components/stepforms/WidgetCheckbox.vue';
+
+describe('Widget Checkbox', () => {
+  it('should instantiate', () => {
+    const wrapper = shallowMount(WidgetCheckbox);
+    expect(wrapper.exists()).to.be.true;
+  });
+
+  it('should have a label', () => {
+    const wrapper = shallowMount(WidgetCheckbox, { propsData: { label: 'test' } });
+    const labelWrapper = wrapper.find('label');
+    expect(labelWrapper.text()).to.equal('test');
+  });
+
+  it('should toggle the right class on click', async () => {
+    const wrapper = mount(WidgetCheckbox, {
+      propsData: { value: false },
+    });
+    expect(wrapper.classes()).not.to.contain('widget-checkbox--checked');
+    wrapper.trigger('click');
+    await Vue.nextTick();
+    expect(wrapper.classes()).to.contain('widget-checkbox--checked');
+    wrapper.trigger('click');
+    await Vue.nextTick();
```

Comment:
I'm not sure that is a good practice to use Vue to handle lifecycle. We should check `vue-test-utils` code but the simplier would be to write something like `wrapper.vm.$nextTick()`

Reply:
You're right, will use localVue